ConwaysGameOfLife/Validator.cs

```diff
@@ -5,14 +5,45 @@ namespace ConwaysGameOfLife
     public class Validator
     {
         // validates input
-        public bool IsUniverse(Universe input)
+
+        public static bool ParseInput(string numberInput)
+        {
+            if (Int32.TryParse(numberInput, out var number))

```

Comment:
can use a discard here to make it clear that we're not using the `out var` at all
```suggestion
            if (Int32.TryParse(numberInput, out var _))
```

Reply:
TryParse returns true if it can parse successfully, and false if it cannot https://docs.microsoft.com/en-us/dotnet/api/system.int32.tryparse?view=net-5.0
Alternative options:
1. assign the Int.TryParse result to a variable and return the variable,
2. return it directly
3. call tryparse where you want to use it and don't have the ParseInput method at all